site/book/06-deploying-packages/03-handling-dependencies.md

```diff
@@ -0,0 +1,74 @@
+Sometimes resources within a package have dependencies that require
+one resource to be applied and reconciled before the next resource
+is applied. For example, a package that includes both Wordpress and
+MySQL might require that MySQL is running before Wordpress is started.
+
+In kpt, this is supported by declaring dependencies with the 
+`config.kubernetes.io/depends-on` annotation.
+
+If you look at the `wordpress-with-dependencies` package, a modified
+version of the `wordpress` package used earlier, you can see that the
+resources belonging to wordpress has the `depends-on` annotation
+referencing the MySQL `StatefulSet`:
+
+```yaml
+# wordpress-with-dependencies/deployment/deployment.yaml (Excerpt)
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wordpress
+  namespace: default
+  labels:
+    app: wordpress
+  annotations:
+    config.kubernetes.io/depends-on: apps/namespaces/default/StatefulSet/wordpress-mysql
+```
+
+The syntax for the resource references are `<group>/namespaces/<namespace>/<kind>/<name>`
+for namespaced resources and `<group>/<kind>/<name>` for cluster-scoped resources.
```

Comment:
Does version not matter?

Reply:
Different versions all point to the same resource. So version doesn't matter here.